### ISSUE TITLE
Daylight saving time

### ISSUE THREAD

**Issue by enriquemorenotent** [simonbaird/CoolClock#15]
Hi, I just realized that the clock for Berlin is still showing the winter time, but last week it should have changed to summer time.

Is there any way to fix this?

**Comment by simonbaird** [comment]
Not really. CoolClock is not timezone aware. You have to adjust the GMT
offset as required.

On Mon, Apr 8, 2013 at 11:51 PM, Enrique Moreno Tent <
notifications@github.com> wrote:

> Hi, I just realized that the clock for Berlin is still showing the winter
> time, but last week it should have changed to summer time.
> 
> Is there any way to fix this?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/simonbaird/CoolClock/issues/15
> .

## 

simon.baird@gmail.com

**Comment by henrahmagix** [comment]
The clock gets its time from the System clock, and applies an offset based on the `gmtOffset` option. Therefore any Daylight Savings Time (DST) will come from the System clock and its settings.
